### DISCUSSION TITLE
How to reset usage of a client?

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [wg-easy/wg-easy#1338] [Q&A] [answered]
How to reset usage of a client?

**Comment by eandersons** [comment, marked as answer]
As far as I know, there is not such feature, but below are a couple workarounds (none of them are quite good, but nonetheless):

- recreate (remove and create; probably restart will do as well) WireGuard Easy container (this will reset stats for all clients);
- remove and re-add client in WireGuard Easy interface (obviously an user will have to import Wireguard settings again, therefore this is not practical).

Such feature will be possible when usage data is stored in database.
